content/strategy-goals/strategy/cody/index.md

```diff
@@ -1,3 +1,58 @@
 # Cody strategy
 
-This is WIP
+## Quicklinks
+
+- [Metrics](https://analytics.amplitude.com/sourcegraph/dashboard/4n7yl67)
+- [Cody FAQ](../../../departments/engineering/teams/cody/about-cody-faq.md)
+- [Cody stories](https://docs.google.com/document/d/1LD6t01TdxPT1LSbzmS2FN_a8tifhBiUOn9YpzJKORWo)
+
+## Vision
+
+Developers can find, understand, write, improve and seure code 100x faster thanks to the power of AI that knows about their company's code, systems, brest practices and policies. Basic, repetitive tasks are gone, and time can be spent on designing and building high value software. Any developer can adopt AI for code because there are guardrails to make it secure and reliable even for the largest regulated enteprises.
```

Comment:
```suggestion
Developers can find, understand, write, improve, and secure code 100x faster thanks to the power of AI which knows about their company's code, systems, best practices, and policies. Basic, repetitive tasks are gone, and time can be spent designing and building high-value software. Any developer can adopt AI for code because there are guardrails to make it secure and reliable even for the largest regulated enterprises.
```